website/src/pages/roadmap.md

```diff
@@ -24,59 +24,57 @@ down by areas on our [stack](/docs/hudi_stack).
-| Support of new indexes in Presto Hudi Connector         | Presto Release / Q3 | [HUDI-4394](https://issues.apache.org/jira/browse/HUDI-4394), [HUDI-4552](https://issues.apache.org/jira/browse/HUDI-4552) |
-| MDT support in Trino Hudi Connector                     | Trino Release / Q2 | [HUDI-2687](https://issues.apache.org/jira/browse/HUDI-2687) |
-| Support of new indexes in Trino Hudi Connector          | Trino Release / Q3 | [HUDI-4394](https://issues.apache.org/jira/browse/HUDI-4394), [HUDI-4552](https://issues.apache.org/jira/browse/HUDI-4552) |
+| Default Java 17 support 	                              | 1.2.0	         | [#16082](https://github.com/apache/hudi/issues/16082)                                                                                                                             |
+| Spark datasource V2 read                                | 1.2.0          | [#15292](https://github.com/apache/hudi/issues/15292)                                                                                                                             |
+| Simplification of engine integration and module organization | 1.2.0          | [#17044](https://github.com/apache/hudi/issues/17044) |
+| End-to-end DataFrame write path on Spark                | 1.2.0          | [#16846](https://github.com/apache/hudi/issues/16846), [#15433](https://github.com/apache/hudi/issues/15433) |
+| Support Hudi 1.0 release in Presto Hudi Connector       | Presto Release / Q2 | [#14992](https://github.com/apache/hudi/issues/14992) |
+| Support of new indexes in Presto Hudi Connector         | Presto Release / Q3 | [#15246](https://github.com/apache/hudi/issues/15246), [#15319](https://github.com/apache/hudi/issues/15319) |
+| MDT support in Trino Hudi Connector                     | Trino Release / Q2 | [#14906](https://github.com/apache/hudi/issues/14906) |
+| Support of new indexes in Trino Hudi Connector          | Trino Release / Q3 | [#15246](https://github.com/apache/hudi/issues/15246), [#15319](https://github.com/apache/hudi/issues/15319) |
 
 ## Platform Components
 
 | Feature                                                                                           | Target Release | Tracking                                                                                                                               |
 |---------------------------------------------------------------------------------------------------|----------------|----------------------------------------------------------------------------------------------------------------------------------------|
-| Syncing as non-partitoned tables in catalogs             | 1.2.0          | [HUDI-9503](https://issues.apache.org/jira/browse/HUDI-9503) |
+| Syncing as non-partitoned tables in catalogs             | 1.2.0          | [#17045](https://github.com/apache/hudi/issues/17045) |
```

Comment:
@yihua the original jira "Split batch calls for getting partition information". is it the right one? if not, pls provide the correct jira

Reply:
The attached JIRA/issue is incorrect.  Let me find the right one.